app/assets/stylesheets/task.scss

```diff
@@ -39,21 +41,75 @@ td {
   padding: 5px;
 }
 
-.task:link {
-  text-decoration: none;
-  color: #222a3f;
+footer {
+  margin-top: 50px;
+  text-align: center;
+  font-size: 20px;
 }
 
-.task:hover {
-  text-decoration: underline;
+.task_link {
```

Comment:
Per CSS naming conventions, we want to use the `-` between words rather than the `_` like we use in ruby